packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -31,22 +31,22 @@ class TextStyle {
   /// The color to use when painting the text.
   final Color color;
 
-  /// The name of the font to use when painting the text.
+  /// The name of the font to use when painting the text (e.g., Roboto).
   final String fontFamily;
 
   /// The size of gyphs (in logical pixels) to use when painting the text.
   final double fontSize;
 
-  /// The font weight to use when painting the text.
+  /// The outline thickness to use when painting the text (e.g., bold).
```

Comment:
(this applies to your dart:ui one as well, and to "fontStyle" below and there):

"outline" might be confused with the kind of font that a lot of people like to use where the glyphs have edge highlights (for lack of another better term). Maybe "typeface thickness"?